storm-core/src/jvm/org/apache/storm/topology/WindowedBoltExecutor.java

```diff
@@ -284,6 +298,10 @@ public void cleanup() {
 
     @Override
     public void declareOutputFields(OutputFieldsDeclarer declarer) {
+        String lateTupleStream = (String) getComponentConfiguration().get(Config.TOPOLOGY_BOLTS_LATE_TUPLE_STREAM);
+        if (lateTupleStream != null) {
+            declarer.declareStream(lateTupleStream, new Fields("late_tuple"));
```

Comment:
Can you make "late_tuple" a constant and put a note in the javadoc of `withLateTupleStream` and Windowing.md that tuple can be accessed via the field "late_tuple" ?